.github/workflows/build.yaml

```diff
@@ -171,3 +171,46 @@ jobs:
 
       - name: flake8
         run: python3 -m flake8
+
+  build_plenum_release:
+    name: Indy Plenum Release
+    needs: [indy_plenum, indy_plenum_tests, lint]
+    runs-on: ubuntu-20.04
+    container:
+      image: ghcr.io/${{ needs.workflow-setup.outputs.GITHUB_REPOSITORY_NAME }}/plenum-build
```

Comment:
To use `needs.workflow-setup.outputs.GITHUB_REPOSITORY_NAME`, you will have to add `workflow-setup` to the `needs:` list.

Reply:
Done